contrib/tester-progs/go/getcpu/getcpu.go

```diff
@@ -18,5 +19,6 @@ func main() {
 		uintptr(unsafe.Pointer(&node)),
 		0,
 	)
+	time.Sleep(1 * time.Second)
```

Comment:
may I ask why you needed this?

Reply:
This is because user stack trace capturing and addresses resolving are async. From bpf we get only linear address and we need some time to resolve them in user space. During this, target process may be finished and we arn't able to read proc/<pid>/maps. So, I put 1 sec sleep in the test because it  finishes too fast to be captured.